lib/rig_inbound_gateway_web/cors.ex

```diff
@@ -0,0 +1,29 @@
+defmodule RigInboundGatewayWeb.Cors do
+  @moduledoc false
+
+  defmacro __using__(which) when is_atom(which) do
+    apply(__MODULE__, which, [])
+  end
+
+  def cors do
+    quote do
+      defp with_allow_origin(conn) do
+        %{cors: origins} = config()
+        put_resp_header(conn, "access-control-allow-origin", origins)
+      end
+    end
+  end
```

Comment:
imho this should always be included. in other words, `use Cors` should import that function; only `preflight_put` should be optional